controllers/kuadrant_controller.go

```diff
@@ -167,6 +179,17 @@ func (r *KuadrantReconciler) Reconcile(eventCtx context.Context, req ctrl.Reques
 		return statusResult, nil
 	}
 
+	if r.isConsolePluginAvailable() {
+		logger.Info("ConsolePlugin API is available, proceeding to apply manifest")
+		if err := r.applyManifest(ctx, "bundle/manifests/kuadrant-console-plugin.yaml", r.Client()); err != nil {
```

Comment:
I am afraid, this is not gonna work 

I see four approaches:
* The kuadrant operator binary has embedded the manifests. (we use this approach for the grafana dashboards on the 3scale operator)
* The kuadrant operator reads from a configmap
* The kuadrant operator has the manifests available in the docker image, so the container can read the file from local filesystem as you try there. 
* The kuadrant operator creates deployment, service, nginx configmap... So all those resources are encoded in golang code. Pretty much like the limitador operator does with limitador. 

Each approach has pro's & con's. Maybe a combination of some of them. IDK.  I would go first with the fourth approach. But let's discuss.